e2e/fixtures/ssr-basic/waku.config.ts

```diff
@@ -4,7 +4,6 @@ import { defineConfig } from 'waku/config';
 
 export default defineConfig({
   vite: {
-    // FIXME How can we limit this config for only mode=development?
     optimizeDeps: {
       exclude: ['@ai-sdk/rsc'],
     },
```

Comment:
The whole purpose of `optimizeDeps` is only for vite, so this is a correct configuration.